spec/controllers/idv/confirmations_controller_spec.rb

```diff
@@ -179,18 +179,6 @@ def index
           )
         end
       end
-
-      context 'ial2 step indicator disabled' do
```

Comment:
Should we flatten (unwrap) the above context "ial2 step indicator enabled" ?

Reply:
Dang it! I'll throw up another PR...